Cargo.lock

```diff
@@ -5236,7 +5236,7 @@ dependencies = [
 
 [[package]]
 name = "penumbra-parameter-setup"
-version = "0.1.0"
+version = "0.80.8"
```

Comment:
ah, sorry about that!

Reply:
Initially I tried to set the version to `0.80.8` explicit, but that didn't resolve the problem. Needed to set `version = { workspace = true }` and then things were happy again.